ingestion/src/metadata/utils/fqn.py

```diff
@@ -554,7 +554,7 @@ def split_table_name(table_name: str) -> Dict[str, Optional[str]]:
     """
     # Revisit: Check the antlr grammer for issue when string has double quotes
     # Issue Link: https://github.com/open-metadata/OpenMetadata/issues/8874
-    details: List[str] = split(table_name.replace('"', ""))
+    details: List[str] = split(table_name.replace('"', "").replace("`", ""))
```

Comment:
Can you share why you added this character replacement? The fqn usually handles special characters without problems.

Reply:
I couldn't figure out whether the backticks that we use to wrap our BigQuery tables were handled in the FQN fucntion when I was initially debugging the ingestion. I will confirm when I am working on setting up the test file whether this is being handled by the fqn

Reply:
From what I could test, fqn.split() doesn't strip the backticks, however ES might be able to find the table regardless of their inclusion when the fqn is built. I will re-run the ingestion without this to confirm whether it was a red herring

Reply:
Seems fine without it, I've dropped the `replace`